### ISSUE TITLE
No filament extrusion after restarting paused print

### ISSUE THREAD

**Issue by maeldupontMX10** [ThomasToka/MarlinFirmware#13]
Hello,

I'am on Ender 3 S1 Plus. I installed your firmware on the 4th of June. It tried pausing a print the clean it a little bit but when I restarted the print, the head was moving following the Gcode but there was no more filament extrusion. The visible gear of the sprite extruder was not spining anymore. It did paused prints in the past with the stock firmware without issues.

Firmware S1 PLUS F4 ABL 5x5

Thank you

**Comment by ThomasToka** [comment]
Hello,

how you started the print? SD print or Host print from for example Octorpint?
How you paused the print? Over touch or over host?
How you resumed print? Over touch or over host?

**Comment by maeldupontMX10** [comment]
Hello,

I started the print via an SD card (I do not have an Octoprint or something similar). And I paused and resumed the print via the touch screen.

P.S. I installed both yout mainboard and Touch screen  firmware.

Thank you for your help

**Comment by ThomasToka** [comment]
Sorry to say but i cant reproduce this. Exactly this works here.

If it is reproducable please make a short video of it.

**Comment by maeldupontMX10** [comment]
I will try reproducing it. Thank you

**Comment by ThomasToka** [comment]
and what i have seen: as the cable goes to park position, users having a brick already but not noticed yet.. notice this on the park position cause the ribbon is bowed there most and so the brick comes into play..

you can also come to my discord. there are many users sharing their stuff..

**Comment by maeldupontMX10** [comment]
okay, I tried reproducing it. Couldn't do it. Itried directly after pausing and after leting the machine cool down but did not reproduce. When the error happend the machine had stopped the heating bed and when I resumed the hotend when to pre-heat temperature (150°C for me) which did not happend again either. stranger behaviour but it migh be my SD card as well (even tho it is a fast SanDsik and not a no name card). Thanks a lot for your time. 

P.S. I printed 3 cable holders in the past to prevent the ribbon form going bad so I think it did not come from there.

I'll join your discord ;)
